src/index.html

```diff
@@ -6,8 +6,498 @@
+            </li>
+            <li class="nav__item">
+              <a href="" class="nav__link">Pricing</a>
+            </li>
+            <li class="nav__item">
+              <a href="" class="nav__link">Blog</a>
+            </li>
+          </ul>
+
+          <a href="" class="menu__call menu__call--nav">
+            +00 123 456 789
+          </a>
+
+          <a href="tel:+00 123 456 789" class="button button--nav">
+            Contact us
+          </a>
+        </div>
+      </div>
+    </nav>
+
+    <main class="main">
+      <div class="container container--about">
+        <p class="main__subtitle">Trusted by forward-thinking software teams around the world</p>
+
+        <div class="main__logo">
+          <img src="./images/logo-1.svg" alt="">
+
+          <img src="./images/logo-2.svg" alt="">
+
+          <img src="./images/logo-3.svg" alt="">
+
+          <img src="./images/logo-4.svg" alt="">
+
+          <img src="./images/logo-5.svg" alt="">
+
+          <img src="./images/logo-6.svg" alt="">
+
+          <img src="./images/logo-7.svg" alt="">
+        </div>
+
+        <h1 class="page__title page__title--main">
+          Whatever work you do,
+          <br>
+          we're able to help
+        </h1>
+
+        <img
+        src="./images/Graphic-main.png"
+        alt=""
+        class="main__photo">
+
+        <div class="grid grid--desktop main__text">
+          <div class="grid__item grid__item--9-12">
+            <div class="main__text-block">
+              <div class="main__text-block-title">
+                <img
+                  src="./images/Icon__main-1.svg"
+                  alt=""
+                  class="main__text-photo">
+
+                <h3 class="main__text-title">
+                  Design and Assets
+                </h3>
+              </div>
+
+              <p class="main__text-subtitle">
+                Make your brand stand out with pixel-perfect design crafted to perfection.
+              </p>
+            </div>
+          </div>
+
+          <div class="grid__item grid__item--9-12">
+            <div class="main__text-block">
+              <div class="main__text-block-title">
+                <img
+                  src="./images/Icon__main-2.svg"
+                  alt=""
+                  class="main__text-photo">
+
+                <h3 class="main__text-title">
+                  Easy Content
+                </h3>
+              </div>
+
+              <p class="main__text-subtitle">
+                Import your demos or build pages visually. Bonus: All images and illustrations included!
+              </p>
+            </div>
+          </div>
+
+          <div class="grid__item grid__item--9-12">
+            <div class="main__text-block main__text-block--last">
+              <div class="main__text-block-title">
+                <img
+                  src="./images/Icon__main-3.svg"
+                  alt=""
+                  class="main__text-photo">
```

Comment:
```suggestion
                <img
                  src="./images/Icon__main-3.svg"
                  alt=""
                  class="main__text-photo"
                >
```
Поправь форматирование